doc/classes/RandomNumberGenerator.xml

```diff
@@ -5,6 +5,14 @@
 	</brief_description>
 	<description>
 		RandomNumberGenerator is a class for generating pseudo-random numbers. It currently uses PCG32. The underlying algorithm is an implementation detail. As a result, it should not be depended upon for reproducible random streams across Godot versions.
+		To generate a random float number (within a given range) based on a time-dependant seed:
+		[codeblock]
+		const MIN_RANGE = float(-10)
```

Comment:
I don't think it's worth adding constants in this example, as it's a matter of taste/convention whether to use constants for this, but it's not really relevant to the actual API usage.

Additionally, floats should be written `-10.0`, not with casting an int to float.

Finally, `RandomNumberGenerator.new()` is a random number *generator*, not a random number itself. And the last statement does not save the generated random number in any variable, so since the RNG is named `random_number` users might think that this `random_number` variable now holds the random number.

So I would make this:
```
var rng = RandomNumberGenerator.new()
rng.randomize()
var random_number = rng.randf_range(-10.0, 10.0)
```

Reply:
Thanks for the suggestions! Yeah, I guess I should have explicitly defined the actual random number result to be a var, like you mention.